### PR TITLE
Fixes 35571 - ACS UI review findings part 2

### PR DESCRIPTION
**Pull request by sjha4**
#### What are the changes introduced in this pull request?
Check review doc for items highlighted in green.

### CODE REVIEW

**Review by jeremylenz — COMMENTED** [review]
Haven't tested yet; just one thought below

**Review by jeremylenz — COMMENTED** [review]
If I select "None" for Credentials in the wizard, shouldn't the SSL CA Certificate dropdown be disabled?

From the document:

"Kebab menu within row should have an space on the right side [2]" - I am not sure what this means. Table row looks fine to me :man_shrugging: 

"Not a good practice to use exclamation marks in notification. Add normal period." Replaced with actual backend error :heavy_check_mark: 

"Row needs to be visibly selected" - :grey_question:   If I click a table row, I expected that row to be selected. But I have to click the link / name of the ACS. Thoughts?

"I want to see new ACS without being forced to refresh manually" :heavy_check_mark: 

"It should be skeleton" :heavy_check_mark: 

"Title should match navigation item’s capitalization" - :heavy_check_mark: 

"Get rid of “Show” and “Hide” - caret should be enough" :heavy_check_mark:

**Review by jeremylenz — COMMENTED** [review]
When deleting the last ACS, I'm getting a blank error toast:
Alternate-Content-Sources

It only seems to happen when the 2-pane view is not active.

**Comment by sjha4** [comment]
>  I really want to add things by double-clicking them

Ya, the patternfly behavior is single click selects and then you can keep selecting multiple. Double click unselects the selection. I guess it makes it easier when you have multiple items in the list to have that behavior. 

> If I click a table row, I expected that row to be selected. But I have to click the link / name of the ACS. Maybe the row should be selected if you click anywhere on the row. Thoughts?

I updated that in this PR. Earlier it was a onClick on the entire data cell but I turned it into a clickable link for name to expand. I dont know which is more natural. I don't think we should do it for the entire row because there are tasks links in the last column and then the actions dropdowns. So we'll have competing onClick actions going on.

File: webpack/components/Loading.js [hunk 1]
```diff
@@ -6,31 +6,41 @@ import {
...
-const Loading = ({ size, showText, loadingText }) => (
-  <Bullseye>
-    <EmptyState>
-      <EmptyStateIcon size={size} variant="container" component={Spinner} />
-      {showText && (
+const Loading = ({
+  size, showText, loadingText, skeleton,
+}) => {
+  if (skeleton) {
+    return <Skeleton height="100%" />;
```
[the remark this answers is not in the document]
Comment (sjha4) [reply webpack/components/Loading.js:17]: Ya..it's a wrapper around the PF4 skeleton..I don't know if we really need that TBH..The PF component seems enough to me for this usecase.

**Review by jeremylenz — COMMENTED** [review]
I'm still seeing the blank error toast after this update.. :confounded:

**Review by jeremylenz — APPROVED** [review]
Works now! Thanks @sjha4 

ACK :+1:

**Pull request merged**